README.md

```diff
@@ -10,8 +10,8 @@ To use ExDoc in your Mix projects, first add ExDoc as a dependency:
 
 ```elixir
 def deps do
-  [{:earmark, "~> 0.1", only: :dev},
-   {:ex_doc, "~> 0.7", only: :dev}]
+  [{:earmark, "~> 0.1.17 or ~> 0.2", only: :dev},
```

Comment:
It can be "~> 0.2". We should push people to the latest anyway. We just support older versions for convenience.